lib/guardian/sidebar_guardian.rb

```diff
@@ -7,6 +7,7 @@ def can_create_public_sidebar_section?
 
   def can_edit_sidebar_section?(sidebar_section)
     return @user.staff? if sidebar_section.public?
+    return @user.staff? if sidebar_section.section_type
```

Comment:
Just noticed this, `staff?` allows both moderator and admins. I believe in this case only admins should have the power to edit?